src/test/java/uk/gov/hmcts/reform/unspec/handler/callback/RequestExtensionCallbackHandlerTest.java

```diff
@@ -155,7 +141,7 @@ void shouldUpdateResponseDeadlineToProposedDeadline_whenExtensionAlreadyAgreed()
             var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
 
             assertThat(response.getData())
-                .containsEntry(RESPONSE_DEADLINE, proposedDeadline.atTime(MID_NIGHT));
+                .containsEntry(RESPONSE_DEADLINE, proposedDeadline.atTime(MID_NIGHT).toString());
```

Comment:
suspicious toString here

Reply:
assertion fails as in `response.getData()` map it's type is String and we have `LocalDateTime` object, so have to make both of same type.